amlip_cpp/test/unittest/node/statusNodeTest.cpp

```diff
@@ -115,12 +115,7 @@ TEST(StatusNodeTest, process_status_parent)
         [&data_arrived, &waiter, &parent_id](const types::StatusDataType& data)
         {
             data_arrived.push_back(data);
-
```

Comment:
Why removing this? I understand it is not needed anymore, but it is not wrong either